alws/utils/pulp_client.py

```diff
@@ -31,6 +31,13 @@ async def create_build_rpm_repo(self, name: str) -> str:
         distro = await self.create_rpm_distro(name, repo_href)
         return distro, repo_href
 
+    async def modify_repository(self, repo_from: str, repo_to: str,
+                                content: str):
+        ENDPOINT = f'{repo_to}modify/'
+        payload = {content: [repo_from]}
+        response = await self.make_post_request(ENDPOINT, data=payload)
+        return response
```

Comment:
We can make this function more generic:
* Remove `content` and `repo_from` arguments
* Add optional arguments `add` and `remove`, which should be None by default, but can be `typing.List[str]`
* Check if this arguments set and add `(add|remove)_content_units` arguments to `payload`

This way you'll make it easier for next person who wants to use this api